src/main/java/net/imglib2/type/numeric/complex/AbstractComplexType.java

```diff
@@ -189,9 +190,9 @@ public boolean equals( final Object obj )
 	@Override
 	public int hashCode()
 	{
-		final int hash1 = Double.hashCode( getRealDouble() );
-		final int hash2 = Double.hashCode( getImaginaryDouble() );
-		return Util.combineHash( hash1, hash2 );
+		final int iHash = Double.hashCode( getRealDouble() );
+		final int rHash = Double.hashCode( getImaginaryDouble() );
```

Comment:
You probably want to swap these, so that `rHash <= getRealDouble()` and `iHash <= getImaginaryDouble()`, no?

Reply:
done